src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A class representing a deadline task.
+ */
+public class Deadline extends Task {
+    private static final DateTimeFormatter DATE_TIME_FORMAT = DateTimeFormatter.ofPattern("MMM dd yyyy");
+    protected String by;
```

Comment:
Shall we make it private to keep it simple for now as no classes currently extends `Deadline`

Reply:
Agreed. I shall change it.